plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -909,8 +909,13 @@ func (c *PodAffinityChecker) InterPodAffinityMatches(pod *v1.Pod, meta interface
 // TODO: Do we really need any pod matching, or all pods matching? I think the latter.
 func (c *PodAffinityChecker) anyPodMatchesPodAffinityTerm(pod *v1.Pod, allPods []*v1.Pod, node *v1.Node, term *v1.PodAffinityTerm) (bool, bool, error) {
 	matchingPodExists := false
+	selector, err := metav1.LabelSelectorAsSelector(term.LabelSelector)
+	namespaces := priorityutil.GetNamespacesFromPodAffinityTerm(pod, *term)
+	if err != nil {
```

Comment:
Please handle err next to the place where you got it (i.e. move this one line above)